modules/account/controllers/profile.classic.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+* @package   test
+* @subpackage account
+* @author    test
+* @copyright 2019 Laurent Jouanneau and other contributors
+* @link      
+* @license    All rights reserved
+*/
+
+use Jelix\Authentication\Account\Manager;
+use Jelix\Authentication\Account\Account;
+
+class profileCtrl extends jController {
+
+    function index() {
+        $rep = $this->getResponse('html');
+
+        $currentUser = Manager::getCurrentUserAccount();
+        $tpl = new \jTpl();
+        if ($currentUser) {
```

Comment:
instead of using a simple array, you could use a form and template plugin to display value instead of displaying the array.

See the [formdata and ctrl_value plugins](https://docs.jelix.org/fr/manuel/composants/jforms/affichage-donnees)

The advantage is that if the developer overload the dao and the form to add fields, it will still works and display automaticcally all additionnals fields